GREEDY.UnitTests/OCRControllerUnitTest.cs

```diff
@@ -1,56 +1,38 @@
 ﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
 using GREEDY.Models;
-using System.Drawing;
-using System;
-using System.Collections.Generic;
 using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 using System.IO;
 using GREEDY.Extensions;
 
-using Emgu;
-using Emgu.CV;
-using Emgu.CV.Structure;
-using Emgu.CV.OCR;
-
-
 namespace GREEDY.UnitTests
 {
     [TestClass]
     public class OCRControllerUnitTest
     {
-        [TestMethod]
+        /*[TestMethod]
         public void UseOCRTest()
         {
             //arrange
             Receipt receipt = new Receipt();
-            var img = new Bitmap("../../TestData/Receipt/test1/original.jpg");
-            var ocr = new Controllers.OCRController();
             var receiptText = File.ReadAllLines("../../TestData/Receipt/test1/originalData.txt").ToList();
             //act
-            receipt = new Controllers.OCRController().UseOCR(img);
+            receipt = new Controllers.OCRController().UseOCR("../../TestData/Receipt/test1/original.jpg");
```

Comment:
Is this a unit test since it uses an external file?

Reply:
Don't really know.. We don't have enough knowledge about Unit tests, maybe someone can look into that more deeply?

Reply:
I guess you are right. I started to learn about unit testing yesterday and will finish it tomorrow. So, now we can push it  and later change if it is not correct.